OrbitLinuxTracing/PerfEventOpen.h

```diff
@@ -93,6 +93,12 @@ static constexpr uint64_t SAMPLE_REGS_USER_ALL =
 static constexpr uint64_t SAMPLE_REGS_USER_SP_IP =
     (1lu << PERF_REG_X86_SP) | (1lu << PERF_REG_X86_IP);
 
+// This must be in sync with struct perf_event_uret_probe in
+// PerfEventRecords.h.
+static constexpr uint64_t SAMPLE_REGS_URET_PROBE = (1lu << PERF_REG_X86_SP) |
+                                                   (1lu << PERF_REG_X86_IP) |
+                                                   (1lu << PERF_REG_X86_AX);
```

Comment:
Move PERF_REG_X86_AX before PERF_REG_X86_SP to reflect the order in the perf_event_open record.